TODO/Android-Studio-Tips.md

```diff
@@ -1,832 +1,542 @@
 > * 原文链接: [Android Studio Tips by Philippe Breault](https://github.com/pavlospt/Android-Studio-Tips-by-Philippe-Breault/wiki)
 * 原文作者 : [Philippe Breault](https://github.com/pavlospt)
 * 译文出自 : [掘金翻译计划](https://github.com/xitu/gold-miner)
-* 译者 : 
+* 译者 : [Jaeger](https://github.com/laobie), [Brucezz](https://github.com/brucezz)
 * 校对者 :
 * 状态 : 待定
 
-Welcome to the Android-Studio-Tips-by-Phillipe-Breault wiki!
+欢迎来到Phillipe Breault发布的Android Studio技巧wiki页面。
 
-Ι am creating this Repository because i think every Android Studio Tip that Phillipe Breault is posting **NEEDS** to be recorded :) 
+我创建了这个仓库是因为我认为Phillipe Breault发布的每一个Android Studio技巧都应该被记录下来。
 
-I will keep updating this repository with every new Tip.
+随着新技巧的发布，我将会一直保持更新。
 
-Stay tuned !!
+敬请关注！！
 
-Credits to: [Philippe Breault](https://plus.google.com/u/0/+PhilippeBreault)
+鸣谢：[Philippe Breault](https://plus.google.com/u/0/+PhilippeBreault)
 
-# Analyze Data flow to Here
 
-This will take the current variable, parameter or field and show you the path it has taken to get here!
+# 1. 分析数据流传递到此处的路径（Analyze data flow to here）
 
-This is very useful when you are entering an unfamiliar place in the code base and you want to understand how this parameter got there.
-
-Additional Tip:
- - There is also the reverse operation "Analyze Data Flow from Here" that will show you where your variable, field or return type will end up.
-
-Shortcut : There is no shortcut for this...
- - Menu: **Analyze → Analyze Data Flow to Here**
- - Find action: **Analyze Data Flow to Here﻿**
+- **描述：**这个操作将会根据当前选中的变量、参数或者字段，分析出其传递到此处的路径。
+当你进入某段陌生的代码，试图明白某个参数是怎么传递到此处的时候，这是一个非常有用的操作。
+- **调用：**Menu → Analyze → Analyze Data Flow to Here
+- **快捷键：**无，可以在设置中指定。
+- **相反的操作：**分析数据流往下传递的路径（Analyze data flow from here），这个将会分析当前选中的变量往下传递的路径，直到结束。
 
 ![](https://lh4.googleusercontent.com/-Fv4MxHWIdHw/VCFWY4Ykv0I/AAAAAAAANoQ/YVe2hmnkAPE/w667-h348-no/31-analyzedataflow.gif)
 
-# Analyze Stacktrace
-
-This will take a stacktrace and make it clickable just as if it had been shown in logcat. 
-This is particularly useful when copying a stacktrace from a bug report or from a terminal.
+# 2. 堆栈追踪分析（Analyze Stacktrace）
 
-Additional tip:
- - You can also analyze a proguarded stack trace using the "ProGuard Unscramble Plugin"
-
-Shortcut : There is no shortcut for this...
- - Menu: **Analyze → Analyze Stacktrace**
- - Find action: **analyze stacktrace﻿**
+- **描述：** 这个操作将会产生一个堆栈追踪，而且和logcat输出中显示的一样，堆栈信息是可以点击的。当你从bug报告中或者终端复制了一份堆栈追踪，使用该操作可以很方便地调试。
```

Comment:
This will take a stacktrace and make it clickable just as if it had been shown in logcat.

这句应该不是“产生一个堆栈追踪”的意思，个人感觉翻译成“这个操作读取一份堆栈追踪信息，并且使它像logcat中那样可以点击”